src/operations/sidebar/SidebarOperations.ts

```diff
@@ -63,4 +65,19 @@ export class SidebarOperations {
   public static hasSubalerts(item: SidebarAlertType): item is SidebarAlertType & { subalerts: SidebarAlertType[] } {
     return 'subalerts' in item;
   }
+
+  public static async fetchCountries(globalRepo: GlobalDataRepository): Promise<SelectedCountry[]> {
+    try {
+      const countryData = await globalRepo.getIpcData(); // TBD: get countries from another request ?
+      return countryData.map((country) => ({
+        name: country.adm0_name,
+        coordinates: {
+          longitude: country.longitude,
+          latitude: country.latitude,
+        },
+      }));
+    } catch {
+      return [];
+    }
+  }
```

Comment:
I think this request is not necessary. The `getMapDataForCountries` method of the global repo is already called on the server, this data includes the countries and their centroids. For now, you can get access to this data by passing it through some components as a prop, but soon a new hook or context can be used which should be much easier.

Also, there are countries that are selectable but don't have ipc data